app/views/trainees/diversity/disclosures/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: "Back",
+    href: trainee_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @disclosure, url: trainee_diversity_disclosure_path(@trainee), method: :put, local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+      
+      <h1 class="govuk-heading-l">Has the trainee shared diversity information?</h1>
+
+      <%= f.govuk_radio_buttons_fieldset(
+        :diversity_disclosure, 
+        legend: { text: "" }, 
```

Comment:
The legend should be the page heading. You should be able use `tag: "h1"` option. Give me shout if you need some help

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/104/commits/01ce1d213a004761f86dc1961b1676983a87e84c